draft-ietf-core-groupcomm-bis.md

```diff
@@ -297,6 +297,33 @@ When this happens, the client normally processes at the CoAP layer each of those
 
 Then, the application is in a better position to decide what to do, depending on the available context information. For instance, it might accept and process all the responses from the same server, even if they are not Observe notifications (i.e., they do not include an Observe option). Alternatively, the application might accept and process only one of those responses, such as the most recent one from that server, e.g. when this can trigger a change of state.
 
+## Caching ## {#sec-caching}
+
+CoAP endpoints that are members of a CoAP group MAY cache responses as intended in Section 5.6 of {{RFC7252}}. In particular, the same rules apply for the set of request options used as "Cache-Key".
+
+Furthermore, building on what defined in Section 8.2.1 of {{RFC7252}}:
+
+   * A client sending a GET or FETCH group request over multicast MAY update a cache with the responses from the servers in the CoAP group. Then, the client uses both cached-still-fresh and new responses as the result of the group request.
```

Comment:
RFC7252 states "Unlike HTTP, the cacheability of CoAP responses does not depend on the request method, but it depends on the Response Code."  If this is the case, why mention GET / FETCH in this section?

Reply:
Wait, it is quoting from RFC7252 - so that's ok.